taskManager/app/controllers/tasks_controller.rb

```diff
@@ -31,13 +32,17 @@ def create
   end
 
   def show
+    @task = Task.find(params[:id])
+    @labels = Label.where(id: label_ids = TaskLabel.where(task_id: @task.id).select(:label_id))
```

Comment:
```suggestion
    label_ids = TaskLabel.where(task_id: @task.id).select(:label_id)
    @labels = Label.where(id: label_ids)
```
あんまりこういう書き方しないので、↑こんな感じが良いと思います